src/utils/filterPeople.ts

```diff
@@ -0,0 +1,17 @@
+import { Person } from '../types/Person';
+
+export function filterPeople(
+  people: Person[],
+  { appliedQuery }: { appliedQuery: string },
+) {
+  let filteredPeople = [...people];
+  const normalizedQuery = appliedQuery.toLocaleLowerCase().trim();
+
+  if (normalizedQuery.length) {
+    filteredPeople = filteredPeople.filter(person =>
+      person.name.toLocaleLowerCase().includes(normalizedQuery),
+    );
+  }
+
+  return filteredPeople;
```

Comment:
method `filter` returns a new array. u don't need the `let` variable here. Avoid creating `let` variables
```suggestion
  const normalizedQuery = appliedQuery.toLocaleLowerCase().trim();

  return normalizedQuery.length 
    ? people.filter((person) => (
      person.name.toLocaleLowerCase().includes(normalizedQuery)
    )) 
    : [...people];
```

Reply:
changed, but prettier removes brackets around the filter callback value